tensorwaves/physics/helicityformalism/kinematics.py

```diff
@@ -12,7 +12,7 @@
 :class:`~SubSystem`.
 """
 import logging
-from typing import List, Optional, Union
+from typing import Dict, List, Optional, Tuple, Union
```

Comment:
I start to prefer:
```python
from typing import (
    Dict,
    List,
    Optional,
    Tuple,
    Union,
)
```
because of diffs and easier sorting (as required by pylint). Note that `black` falls to this format once you wrap this list in parentheses, ending it with a commit.

Reply:
done